services/services.go

```diff
@@ -53,6 +65,9 @@ func NewServiceReplication() ServiceReplication { return new(serviceReplication)
 type ServiceSharding interface {
 	// Len is the count of shards to use as the sharding range
 	Len() int
```

Comment:
NumShards is used in storage
On Sun, Aug 28, 2016 at 7:26 PM Rob Skillington <notifications@github.com>
wrote:

> In services/services.go
> <https://github.com/m3db/m3cluster/pull/15#discussion_r76541424>:
>
> > @@ -53,6 +65,9 @@ func NewServiceReplication() ServiceReplication { return new(serviceReplication)
> >  type ServiceSharding interface {
> >  	// Len is the count of shards to use as the sharding range
> >  	Len() int
>
> Have we used a term in storage for this yet? We should just copy if so.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/m3db/m3cluster/pull/15/files/a88fb37c7bc737f5fc4da6dc2d0f763e130dc93a#r76541424>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAX-WeHOfuz_n4pU9tbkIiYwD22032oFks5qkhkzgaJpZM4JvCWr>
> .
>